Blish HUD/Controls/ColorBox.cs

```diff
@@ -1,114 +1,111 @@
-
-        //        OnPropertyChanged("ColorId");
-        //        OnPropertyChanged();
-        //        this.OnColorChanged?.Invoke(this, EventArgs.Empty);
-        //    }
-        //}
-
-        //private int _colorId;
-        //public int ColorId {
-        //    get { return _colorId; }
-        //    set {
-        //        if (_colorId == value) return;
-                
-        //        Task<DyeColor> aDye = BHGw2Api.DyeColor.GetById(this.ColorId);
-        //        aDye.ContinueWith(clr => {
-        //            if (!clr.IsFaulted) {
-        //                _colorId = value;
-        //                _color = aDye.Result;
-
-        //                OnPropertyChanged();
-        //                OnPropertyChanged("Color");
-        //                this.OnColorChanged?.Invoke(this, EventArgs.Empty);
-        //            }
-        //        });
-        //    }
-        //}
-
-        //#region "Statics (Sprites & Shared Resources)"
-
-        //private static TextureRegion2D[] spriteBoxes;
-        //private static TextureRegion2D spriteHighlight;
-
-        //private static void LoadStatics() {
-        //    if (spriteBoxes != null) return;
-
-        //    // Load static sprite regions
-        //    spriteBoxes = new TextureRegion2D[] {
-        //        Resources.Control.TextureAtlasControl.GetRegion("colorpicker/cp-clr-v1"), Resources.Control.TextureAtlasControl.GetRegion("colorpicker/cp-clr-v2"), Resources.Control.TextureAtlasControl.GetRegion("colorpicker/cp-clr-v3"), Resources.Control.TextureAtlasControl.GetRegion("colorpicker/cp-clr-v4"),
-        //    };
-        //    spriteHighlight = Resources.Control.TextureAtlasControl.GetRegion("colorpicker/cp-clr-active");
-        //}
-
-        //#endregion
-
-        //private readonly int _drawVariation;
-
-        //public ColorBox() : base() {
-        //    LoadStatics();
-
-        //    this.Size = new Point(COLOR_SIZE);
-
-        //    _drawVariation = RandomUtil.GetRandom(0, 3);
-        //}
-
-        //protected override void OnMouseMoved(MouseEventArgs e) {
-        //    base.OnMouseMoved(e);
-
-        //    this.BasicTooltipText = this.Color?.Id.ToString() ?? "None";
-        //}
-
-        //protected override CaptureType CapturesInput() {
-        //    return CaptureType.Mouse;
-        //}
-
-        //protected override void Paint(SpriteBatch spriteBatch, Rectangle bounds) {
-        //    spriteBatch.Draw(spriteBoxes[_drawVariation], bounds, this.Color?.Fur?.Rgb.ToXnaColor() ?? Microsoft.Xna.Framework.Color.White);
-
-        //    if (this.MouseOver || this.Selected)
-        //        spriteBatch.Draw(spriteHighlight, bounds, Microsoft.Xna.Framework.Color.White * 0.7f);
-        //}
-
-    //}
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class ColorBox : Control {
+
+        public event EventHandler<EventArgs> OnColorChanged;
+        public event EventHandler<EventArgs> OnSelected;
```

Comment:
The TODO above is actually about switching this these to use the style `ColorChanged` and `Selected` instead of using the 'On' prefix before them.  Can knock the TODO out and get this swapped in one fix.